src/pages/LandingPage.py

```diff
@@ -31,4 +35,10 @@ def get_upcoming_public_event(self):
         self.scroll_down_page()
         return self.find_element_by_css(self.UPCOMING_PUBLIC_EVENT)
 
+    def click_join_event_link(self):
```

Comment:
Do you need a method for this?
Maybe you can use join_event_link.click_btn_by_xpath() since it's already included in the constructor.

Reply:
I used admin_setup and it doesn't have access to elements in the constructor. I received the error when trying to write like this: admin_setup.landing.join_event_link.click_btn_by_xpath() in my test. Maybe you can advise me how to fix this (without changing all my test)?

Reply:
I understood why I received the error. "admin_setup.landing.join_event_link.click_btn_by_xpath() " didn't work because it didn't close modal window after login so quickly and  I received "ElementNotInteractableException" so I needed to add some waits. I fixed these. Will push soon